x/launch/client/cli/query_vested_account_test.go

```diff
@@ -0,0 +1,172 @@
+package cli_test
+
+import (
+	"fmt"
+	"github.com/tendermint/spn/testutil/sample"
```

Comment:
import sort does not respect the lint style

Reply:
This is not caught with `make lint`

Reply:
I removed the import check because is returning errors regarding the scaffold comments into the imports and it's falling all PR's. I can put it back again and we can add `// nolint` flags into the code, what do you think?